File: ExamPreparation/AnimeList/CSharp Solution/AnimeList/Models/Anime.cs
using System.ComponentModel.DataAnnotations;
using System.Web.Mvc;

namespace AnimeList.Models
{
    public class Anime
    {
        [Key]
        public int Id { get; set; }

        [Required]
        [MinLength(1)]
        public string Name { get; set; }

        [Required]
        public int Rating { get; set; }

        [Required]
        public string Description { get; set; }

        [Required]
        public string Watched { get; set; }
    }
}

File: ExamPreparation/AnimeList/JavaScript Solution/models/Anime.js
const mongoose = require('mongoose');

let animeSchema = mongoose.Schema({
    name: {type: 'string', required: 'true'},
    description: {type: 'string', required: 'true'},
    watched: {type: 'string', required: 'true'},
    rating: {type: 'number', required: 'true'}
});

let Anime = mongoose.model('Anime', animeSchema);

module.exports = Anime;